tests/metrics/estimate_gas.py

```diff
@@ -165,25 +128,29 @@ def estimate_gas(analyzer: AnalyzeGas = None) -> None:
     #
     # Setup
     #
+
     if AnalyzeGas is None:
```

Comment:
Perhaps you mean this?
```suggestion
    if analyzer is None:
```
(This GH suggestion may be buggy, it seems it removes a white-space even if the number of spaces is correct. I'm not sure if the patch is wrong, or only the rendering here)

Reply:
Whoops! :heavy_check_mark: